src/api/app/models/workflow.rb

```diff
@@ -24,6 +38,26 @@ def steps
     end
   end
 
+  def filters
+    filters = {}
+
+    return filters if supported_filters.blank?
+
+    SUPPORTED_FILTERS.each do |filter|
+      SUPPORTED_FILTER_TYPES.each do |filter_type|
+        # The filter type might not be present... so in that case, we go to the next
+        next unless (filter_values = supported_filters.dig(filter, filter_type))
+
+        filters[filter] = { "#{filter_type}": filter_values }
+
+        # As soon as a supported filter type is present, we get out of the loop since the following filter types have a lower precedence
```

Comment:
Are they? Am I not free in the order I write them into the `workflow.yml`?

Reply:
Ah this is the order of `SUPPORTED_FILTER_TYPES`. I find this a bit too implicit.